Assets/Scripts/UI/Camera/IN_GAME_MAIN_CAMERA.cs

```diff
@@ -427,57 +426,63 @@ public void update2()
         }
         if (gametype != GAMETYPE.Stop)
         {
+
             if (this.gameOver)
             {
-                if (InputManager.KeyDown(InputHuman.AttackSpecial))
-                {
-                    if (this.spectatorMode)
-                    {
-                        this.setSpectorMode(false);
-                    }
-                    else
-                    {
-                        this.setSpectorMode(true);
-                    }
-                }
-                if (InputManager.KeyDown(InputHuman.Item1))
+                
+                this.setSpectorMode(true);
+                FengGameManagerMKII.instance.ShowHUDInfoCenter(
```

Comment:
FengGameManagerMKII.instance.ShowHUDInfoCenter( can be removed here, as it does nothing.

Reply:
I tested and this is the one displaying HUD, the duplicate in FengGameManagerMKII doesn't for some reason, and i did remove it there to test as well.

Reply:
On which gamemode did you test it? The second ShowHUDInfo center only gets called when your gamemode is NOT Racing

Reply:
I tested all gamemodes. Removing the HUD statements from FengGameManagerMKII and keeping it here keeps it open, idk why but it does